libtelemetry/src/main/java/com/mapbox/android/telemetry/MapBuildEvent.java

```diff
@@ -1,6 +1,6 @@
 package com.mapbox.android.telemetry;
 
 
-interface MapBuildEvent {
+public interface MapBuildEvent {
```

Comment:
hey @Chaoba thanks for continuing looking into this. A quick note here: can we have all `Map*.java` classes moved to `gl-native` repo instead of exposing them publicly from telem sdk?

Reply:
That sounds good.